dsse/signerverifier.go

```diff
@@ -0,0 +1,9 @@
+package dsse
+
+// SignerVerifer provides both the signing and verification interface.
+type SignerVerifier interface {
+	Signer
```

Comment:
could consolidate the `Signer` and `Verifier` interface here.

Reply:
Do you mean move Signer and Verifier in here as well and have only the EnvelopeSigner and EnvelopeVerifier implementations in sign.go and verify.go?

Reply:
Did that in a separate commit.